src/Products/SiteAccess/VirtualHostMonster.py

```diff
@@ -104,7 +104,7 @@ def set_map(self, map_text, RESPONSE=None):
                 if hostname not in host_map:
                     host_map[hostname] = {}
                 host_map[hostname][port] = pp
-            except 'LineError' as msg:
```

Comment:
@icemac You've added Products.SiteAccess with this commit.
I don't find the original repo anymore and can't do a git blame there.
What's meant with a ``'LineError'``?

Reply:
The code was actually added in f58021f472a0d7490b6913c8168d723f50f269cd 18 years ago. The code raised the `LineError` exception those days. In fb33a42d9bc934c540829ceb9373dda1d76551ac it was changed to `ValueError` so the except statement should be changed, too.